src/migration/tasks/AddDefaultPropertyToTagsTask.ts

```diff
@@ -0,0 +1,50 @@
+import Constants from '../../utils/Constants';
+import Logging from '../../utils/Logging';
+import MigrationTask from '../MigrationTask';
+import { ServerAction } from '../../types/Server';
+import Tenant from '../../types/Tenant';
+import TenantStorage from '../../storage/mongodb/TenantStorage';
+import global from '../../types/GlobalType';
+
+const MODULE_NAME = 'AddDefaultPropertyToTagsTask';
+
+export default class AddDefaultPropertyToTagsTask extends MigrationTask {
+  async migrate(): Promise<void> {
+    const tenants = await TenantStorage.getTenants({}, Constants.DB_PARAMS_MAX_LIMIT);
+    for (const tenant of tenants.result) {
+      await this.migrateTenant(tenant);
+    }
+  }
+
+  async migrateTenant(tenant: Tenant): Promise<void> {
+    // Add the default property to tags
+    const result = await global.database.getCollection<any>(tenant.id, 'tags').updateMany(
```

Comment:
I replaced your multiple updates with only one MongoDB query.
Just keep it in mind